scopes/mdx/readme/readme.env.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@teambit/component';
+import { Environment } from '@teambit/envs';
+import { DocsMain } from '@teambit/docs';
+
+export const ReadmeEnvType = 'readme';
+
+export class ReadmeEnv implements Environment {
+  constructor(private docs: DocsMain) {}
+  private removeDocsDevPatterns(): string[] {
```

Comment:
I would rename this function.
it's not really removing, but generates a negative pattern.
so something like get/generate negateDocsDevPattern